docs/1.concepts/basics/transactions/gas-advanced.md

```diff
@@ -0,0 +1,186 @@
+---
+id: gas-advanced
+title: Gas - Advanced
+
+---
+
+## Costs of complex actions {#costs-of-complex-actions}
+
+The numbers above should give you the sense that transactions on NEAR are cheap! But they don't give you much sense of how much it will cost to use a more complex app or operate a NEAR-based business. Let's cover some more complex gas calculations: deploying contracts and function calls.
```

Comment:
here it talks about "the numbers above", but there's nothing above this paragraph? 🤔